aries-site/src/examples/templates/FilterControls/FilterRangeSelector.js

```diff
@@ -0,0 +1,54 @@
+import { useContext, useState } from 'react';
+import PropTypes from 'prop-types';
+import { Box, FormField, RangeSelector, Stack, Text } from 'grommet';
+
+import { FilterContext } from '.';
+
+export const FilterRangeSelector = ({ attr }) => {
+  const { data, filters, setFilters, getFilterOptions } = useContext(
+    FilterContext,
+  );
+  const { property, header, inputProps } = attr;
+  const options = getFilterOptions(data, property);
+  const [range, setRange] = useState(
+    (filters[property] && filters[property].value) || [
+      inputProps.min || 0,
+      inputProps.max || Math.max(...options),
+    ],
+  );
+
+  return (
+    <Box flex={false}>
+      <FormField htmlFor={property} name={property} label={header}>
+        <Stack>
+          <Box background="border" height="3px" direction="row" />
+          <RangeSelector
+            id={property}
+            name={property}
+            min={inputProps.min || 0}
```

Comment:
Should we spread any other inputProps below, in case user has provided other's that we're not specifically accounting for?
{...inputProps}

Reply:
good call - I like